lib/common/resources.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 the Pacemaker project contributors
```

Comment:
2024

Reply:
Whoops, updated. BTW we recently added `make -C devel copyright` which I obviously forgot to run :)